pages/learn/welcome/security.md

```diff
@@ -7,6 +7,13 @@ excerpt: The {{ $names.company.lower }} approach to securing your fleet
 
 {{ $names.company.upper }} takes your fleet’s security very seriously.  We understand that your data and devices are valuable and often critical, and we take precautions to ensure that your devices and data are not compromised.
 
+## Trust and Transparency
+At {{ $names.company.upper }}, our approach to security is guided by ISO 27001:2022, ensuring a structured, reliable system that aligns with our values. Our ISMS (Information Security Management System) has undergone multiple audits by accredited third parties to verify our compliance and effectiveness. Rather than using rigid templates, we customized our implementation to fit our team's way of working, maintaining transparency without compromising our culture. This system is continually monitored and improved, demonstrating our commitment to providing a secure and dependable platform while staying true to who we are.
```

Comment:
```suggestion
At {{ $names.company.lower }}, our approach to security is guided by ISO 27001:2022, ensuring a structured, reliable system that aligns with our values. Our ISMS (Information Security Management System) has undergone multiple audits by accredited third parties to verify our compliance and effectiveness. 

Rather than using rigid templates, we customized our implementation to fit our team's way of working, maintaining transparency without compromising our culture. This system is continually monitored and improved, demonstrating our commitment to providing a secure and dependable platform while staying true to who we are.
```

When in a sentence, we do `balena`
Broke the paragraph for more readability.